src/components/LeftSidebar/tabs/Skills.js

```diff
@@ -31,7 +32,7 @@ const SkillsTab = ({ data, onChange }) => {
       <hr className="my-6" />
 
       {data.skills.items.map((x, index) => (
-        <Item item={x} key={index} index={index} onChange={onChange} dispatch={dispatch} />
+        <Item item={x} key={x.id} index={index} onChange={onChange} dispatch={dispatch} />
```

Comment:
index prop probably not necessary since now we can use id to find the index. I'll key the index here for now. I'll let you make the call